README.md

```diff
@@ -72,71 +72,122 @@ Features:
+### Fedora Dependencies
 
-Configure your local Jackett instance at [http://localhost:9117](http://localhost:9117).  You'll need to add indexers and copy your api key to enter into nefarious.  Some popular examples are *The Pirate Bay*, *1337x*, *RARBG*.
+Install the Docker repository and update metadata cache
 
-#### Configure Transmission
+    sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
+    sudo dnf makecache
 
-The default download path is `/tmp/transmission`, so make sure to edit the `docker-compose.yml` to change that to something like:
-  
-    /home/bob/Downloads:/downloads
+install docker and docker-compose from repository
+
+    sudo dnf install docker-ce
+    sudo dnf -y install docker-compose
+
+At the moment Docker-Compose doesn't fully work without modification on Fedora 31.  30,29,28, and so on should work however.  If you're running Fedora 31, use the following Reddit Thread and most recent post at your own discretion. 
+https://www.reddit.com/r/Fedora/comments/d8ukd0/has_anyone_managed_to_run_docker_ce_on_fedora_31/
+
+### Windows OS Dependencies
+
+You'll need to ensure that your PC is running a version of Windows 10 64-bit Professional, Education, or Enterprise.  Docker for Windows requires Hyper-V techonology, which is not supported by Windows 10 Home.  You'll also need to ensure that your PC has Virtualization enabled in BIOS before attempting to install Docker for Windows.  While Nefarious is not by any means a Linux exclusive application, it is much easier to setup on either a Linux based OS, or on a Linux Virtual Machine through your preferred VM software on any actively updated version of Windows. Consult appropriate documentation relating to said software if you wish to setup folder shares between your Linux VM and your Windows install.  Docker Toolbox is also an option, as it runs docker commands on non-Hyper-V supported OSes by running them through an integrated Linux VM.  If you'd prefer to avoid using something like Virtualbox, VMWare, or other separate Virtualization software, this would would probably work best for you.  That being said, we'd recommend this only be done by experienced users of Docker software.
+
+### OS Agnostic Setup Guide
+
+### Part 1 - Setup performed from terminal
+
+    sudo systemctl start docker.service
+    sudo systemctl enable docker.service
+    sudo groupadd docker
+   
+If you get an error 'groupadd: group 'docker' already exists', proceed
+
+    sudo usermod -aG docker $YOURUSERHERE$ 
+
+The two above commands allow for Docker commands to be executed without needing to call for sudo each time you run any Docker command.  Logout, then login again.
+
+    docker run --rm -it --init alpine echo "success"   
     
-Where the new download path would be `/home/bob/Downloads`.  Leave the right side alone, i.e `/downloads` as that maps to an internal container path.
+This ensures Tini, a zombie docker process killer, is activated
```

Comment:
I don't see any other reference to `Tini`.  Did you mean to include this?

Reply:
I did, and it was more for the benefit of those curious as to what each particular executed command accomplished.  If it seems unnecessary I'd be happy to remove it.